.github/actionlint.yaml

```diff
@@ -0,0 +1,5 @@
+# This is a bit of a hack to get actionlint to accept larger github runners
+# see https://github.com/rhysd/actionlint/issues/337#issuecomment-1722949069
+self-hosted-runner:
+  labels:
+    - ubuntu-22-04-32-core
```

Comment:
@alecthomas I added this to avoid that `actionlint` error that the PR was getting.

Reply:
Nice!

Reply:
We might be able to remove this, and possibly save money if we get a smaller machine too: https://github.com/rhysd/actionlint/commit/64902ec49b95dbd4e88bf2f543885edb18bbff09#diff-8deffdcd8789dd0eda3f521d38ca69d2e988777fd4d50aec144761900ffcad37

Looks like `ubuntu-latest-4-cores`, `ubuntu-latest-8-cores`, and `ubuntu-latest-16-cores` might work

Reply:
I wonder how you control the number of jobs/machine though. For this one I've explicitly set it to 8, so each job gets 4 cores minimum.